diagnostic.md

```diff
@@ -51,25 +54,25 @@ Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+Because the class inside model inherits from the Activemodel
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
 us?
 
 ```md
-// your response here
+It drops you into the rails command line interface.
 ```
 
 What does typing `rails db` in a directory that houses a rails app do for us?
 
 ```md
-// your response here
+It also drops you into the rails command line interface.
```

Comment:
There's a typo here, so I won't mark either of these incorrect. 

`bin/rails db` opens your rails applications postgres database
`bin/rails console` (not dbconsole) opens your rails CLI

👍